app/helpers/invoices_helper.rb

```diff
@@ -7,9 +7,9 @@ def get_clients
     current_user.company.clients.map{ |client|[client.name,client.id]}
   end
 
-  def get_address(invoice)
-    if invoice.address.present?
-      "#{invoice.address.street_1} #{invoice.address.street_2}, \n#{invoice.address.city}, #{invoice.address.state}, \n#{invoice.address.pincode.to_s}"
+  def get_address(client)
```

Comment:
If this is a common method then the param name should be generic like `record` instead of `client`

Also rename `get_address` with `formatted_address`